clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -443,7 +443,7 @@ static void checkSYCLType(Sema &S, QualType Ty, SourceRange Loc,
   // for example the quad type __float128 will cause errors in the
   // SPIR-V translation phase.
   // Here we check any potentially unsupported declaration and issue
-  // a deferred diagnostic, which will be emitted iff the declaration
+  // a deferred diagnostic, which will be emitted if the declaration
```

Comment:
Was this change intended? I read the original "iff" as "if and only iff" (https://www.merriam-webster.com/dictionary/iff).

Reply:
Oh right. My bad. I didn't think of 'if and only if'. Reverted this change. Thanks!

Reply:
This is a common recurring issue: what about not using abbreviations in comments, considering the price of storage for 1 character nowadays? :-)

Reply:
I'm ok with your suggestion and will keep that in mind for my patches :) However, I'm not sure about enforcing it during review since I'm not sure what clang community's rule is . I suspect we just go along with whatever community standard is. @AaronBallman might know the answer

Reply:
We won't be getting rid of abbreviations in general any time soon (AST, CFG, IR, LLVM, HIP, CUDA, etc), but `iff` is a bit different from those IMO. I looked through the Clang codebase and found about 100 uses of `iff` that all seem to be correct rather than typos (from spot checking, so I might have missed some). Somewhat amusingly `if and only if` appears... about 100 times as well. :-D

I don't have a strong opinion and I doubt community does either. I'm fine leaving this as `iff`, but @elizabethandrews is free to expand it to the long form it iff she thinks it's worthwhile. :-)

Reply:
Since this comment is technically in an area I don't need to touch in this patch, I'm going to leave it as is for now. I'll keep an eye out in the future :)

Reply:
Good comments means empathy, inclusive coding, a social experience...
So it makes sense to have less abbreviations. :-)